#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

time start_time
duration end_time_offset
uint32 flags
uint16 scan_number
ScannerInfo[] scanner_info
ScanPoint[] scan_points 